chapter_notation/index.md

```diff
@@ -1,85 +1,94 @@
+* $\mathbf{1}_{\mathcal{X}}(z)$: the set-membership indicator function, evaluates to $1$ if the element $z$ belongs to the set $\mathcal{X}$ and $0$ otherwise
+* $\mathbf{(\cdot)}^\top$: transpose of a vector or a matrix
+* $\mathbf{X}^{-1}$: inverse of matrix $\mathbf{X}$
 * $\odot$: Hadamard (elementwise) product
-* $[\cdot, \cdot]$: Concatenation
-* $\lvert \mathcal{X} \rvert$: Cardinality of set $\mathcal{X}$
+* $[\cdot, \cdot]$: concatenation
 * $\|\cdot\|_p$: $L_p$ norm
 * $\|\cdot\|$: $L_2$ norm
-* $\langle \mathbf{x}, \mathbf{y} \rangle$: Dot product of vectors $\mathbf{x}$ and $\mathbf{y}$
-* $\sum$: Series addition
-* $\prod$: Series multiplication
-* $\stackrel{\mathrm{def}}{=}$: Definition
+* $\langle \mathbf{x}, \mathbf{y} \rangle$: dot product of vectors $\mathbf{x}$ and $\mathbf{y}$
+* $\sum_i=1^n$: summation over collection with indices ranging from $1$ to $n$
+* $\prod$: product over collection with indices ranging from $1$ to $n$
+* $\stackrel{\mathrm{def}}{=}$: an equality asserted as a definition of the symbol on the left-hand side
 
 
 ## Calculus
 
-* $\frac{dy}{dx}$: Derivative of $y$ with respect to $x$
-* $\frac{\partial y}{\partial x}$: Partial derivative of $y$ with respect to $x$
-* $\nabla_{\mathbf{x}} y$: Gradient of $y$ with respect to $\mathbf{x}$
-* $\int_a^b f(x) \;dx$: Definite integral of $f$ from $a$ to $b$ with respect to $x$
-* $\int f(x) \;dx$: Indefinite integral of $f$ with respect to $x$
+* $\frac{dy}{dx}$: derivative of $y$ with respect to $x$
+* $\frac{\partial y}{\partial x}$: partial derivative of $y$ with respect to $x$
+* $\nabla_{\mathbf{x}} y$: gradient of $y$ with respect to $\mathbf{x}$
+* $\int_a^b f(x) \;dx$: definite integral of $f$ from $a$ to $b$ with respect to $x$
+* $\int f(x) \;dx$: indefinite integral of $f$ with respect to $x$
 
 ## Probability and Information Theory
 
-* $P(\cdot)$: Probability distribution
-* $Z \sim P$: Random variable $Z$ has probability distribution $P$
-* $P(X \mid Y)$: Conditional probability of $X \mid Y$
-* $p(x)$: Probability density function
-* ${E}_{x} [f(x)]$: Expectation of $f$ with respect to $x$
-* $X \perp Y$: Random variables $X$ and $Y$ are independent
-* $X \perp Y \mid Z$: Random variables  $X$  and  $Y$  are conditionally independent given random variable $Z$
-* $\mathrm{Var}(X)$: Variance of random variable $X$
-* $\sigma_X$: Standard deviation of random variable $X$
-* $\mathrm{Cov}(X, Y)$: Covariance of random variables $X$ and $Y$
-* $\rho(X, Y)$: Correlation of random variables $X$ and $Y$
+* $X$: a random variable
+* $P$: a probability distribution
+* $X \sim P$: the random variable $X$ has distribution $P$
+* $P(X=x)$: the probability assigned to the event where random variable $X$ takes value $x$
+* $P(X \mid Y)$: the conditional probability distribution of $X$ given $Y$
+* $p(\cdot)$: a probability density function (PDF) associated with distribution P
+* ${E}[X]$: expectation of a random variable $X$
```

Comment:
So why do we remove f(x) here? It's actually used more often in the book :)

Reply:
because it's redundant f(x) is just a random variable

Reply:
(what's being introduced here is the notation for expectation)

Reply:
How about using ${E}_{X} [f(X)]$ given that $f(X)$ is more general than $X$?

Reply:
no, it's not more general. X can be any random variable, including a function of another random variable.